File: icd0017-2018s/radio/app/src/main/java/com/example/radio2019/Domain/Song.java
package com.example.radio2019.Domain;

public class Song {
    public int id;
    public String songTitle;
    public int timesPlayed;

    public Song(String songTitle, int timesPlayed){
        this(0, songTitle, timesPlayed);
    }

    public Song(int id, String songTitle, int timesPlayed){
        this.id = id;
        this.songTitle = songTitle;
        this.timesPlayed = timesPlayed;
    }
}

File: icd0017-2018s/Radio_iOS/Radio_iOS/Domain/Song.swift
//
//  Song.swift
//  Radio_iOS
//
//  Created by user154645 on 5/11/19.
//  Copyright © 2019 user154645. All rights reserved.
//

import Foundation

class Song {
    var id: Int
    var songTitle: String
    var timesPlayed: Int
    
    convenience init(songTitle: String, timesPlayed: Int){
        self.init(id: 0, songTitle: songTitle, timesPlayed: timesPlayed)
    }
    
    init(id: Int, songTitle: String, timesPlayed: Int){
        self.id = id
        self.songTitle = songTitle
        self.timesPlayed = timesPlayed
    }
    
}
